x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/stats/VerifierMetricsTests.java

```diff
@@ -279,6 +298,9 @@ public void testMvExpand() {
         assertEquals(0, drop(c));
         assertEquals(1L, keep(c));
         assertEquals(0, rename(c));
+
+        assertEquals(1, function("WILDCARDLIKE", c));
```

Comment:
What function is `wildcardlike`? :-)

Reply:
That's what `funcitonName()` returns for our LIKE, that is [WildcardLike](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/string/WildcardLike.java).

Probably we should not rely on `funcitonName()` at all, but rather use the information provided by FunctionRegistry